lib/public/UrlChecker.js

```diff
@@ -66,21 +67,25 @@ UrlChecker.prototype.enqueue = function(url, baseUrl, customData)
 	// Undocumented internal use: enqueue(linkObj)
 	if (typeof url==="object" && !(url instanceof String) && url.broken_link_checker===true)
 	{
-		return this.linkQueue.enqueue(
-		{
-			url: url.url.parsed,
-			data: { customData:customData, linkObj:url }
-		});
+		if (shouldQueryUrl(url.url.parsed.href, this)===true) {
```

Comment:
Done.

Yeah, sorry, I'm trying to follow the style, but I likely will miss some stuff since I'm not used to it yet. Linting might be helpful to ensure that I'm not screwing up accidentally. I assure you that it is not malicious.